packages/preact-iso/lazy.d.ts

```diff
@@ -0,0 +1,5 @@
+import { VNode } from 'preact';
+
+export default function lazy<T>(load: () => Promise<{ default: T }>): T;
+
+export function ErrorBoundary(props: { children?: VNode }): VNode;
```

Comment:
I think children needs to be typed as `ComponentChildren` here. Otherwise this wouldn't be valid:

```jsx
<ErrorBoundary>foo</ErrorBoundary>
```